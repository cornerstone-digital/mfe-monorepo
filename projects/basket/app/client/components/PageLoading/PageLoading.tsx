import Spinner from '@web-core/components/atoms/Loading'

import styles from './PageLoading.scss'

export const PageLoading = () => (
  <div className={styles.wrapper}>
    <Spinner appearance="brand" size="lg" />
  </div>
)

export default PageLoading
