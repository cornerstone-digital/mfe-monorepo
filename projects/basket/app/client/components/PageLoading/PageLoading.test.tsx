import { render } from '@testing-library/react'
import PageLoading from './PageLoading'

describe('<PageLoading />', () => {
  test('should render properly', () => {
    const { asFragment } = render(<PageLoading />)
    expect(asFragment()).toMatchSnapshot()
  })
})
