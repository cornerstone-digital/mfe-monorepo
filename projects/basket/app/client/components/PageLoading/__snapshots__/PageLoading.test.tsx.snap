// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageLoading /> should render properly 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
  >
    <div
      class="wrapper wrapper-left "
    >
      <div
        class="spinner-wrapper spinner-wrapper-left"
      >
        <div
          class="spinner spinner-brand spinner-lg spinner "
        >
          <svg
            viewBox="25 25 50 50"
          >
            <circle
              cx="50"
              cy="50"
              r="20"
            />
          </svg>
          <div
            class="spinner spinner-brand spinner-lg spinner vfuk-LoadingSpinner__fallback_loader"
          >
            <div />
            <div />
            <div />
            <div />
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
